src/Test/L0/Worker/Build/TrackingManagerL0.cs

```diff
@@ -520,5 +520,12 @@ public void UpdatesTrackingConfigJobRunProperties()
                 Assert.True(DateTimeOffset.Now.AddSeconds(1) >= config.LastRunOn);
             }
         }
+
+        //removes CR so the test can run on Linux, even if it was synced on a Windows drive share
```

Comment:
I already merged, but will back out this change. IMO I would use new line constant instead of placing new lines in string literal, which seems fragile. This problem is similar to concatenating paths, which is solved by "Path.Combine".